client/my-sites/marketing/connections/service.jsx

```diff
@@ -112,12 +115,24 @@ export class SharingService extends Component {
 			'must-disconnect' === connectionStatus
 		) {
 			this.removeConnection();
+			this.props.recordTracksEvent( 'calypso_connections_disconnect_button_click', {
+				service: this.props.service.ID,
```

Comment:
In this case if it is destructured then it would be called ID, and so the props parameter would still have to specify `service` as a property. So I decided to just just leave it like this.